app/src/main/java/ar/com/wolox/android/example/utils/UserSession.kt

```diff
@@ -22,4 +22,16 @@ constructor(private val mSharedPreferencesManager: SharedPreferencesManager) {
             field = username
             mSharedPreferencesManager.store(Extras.UserLogin.USERNAME, username)
         }
+
+    var email: String? = null
+        get() {
+            if (field == null) {
+                field = mSharedPreferencesManager.get(Extras.UserLogin.EMAIL, null)
```

Comment:
maybe you can set "" instead of null. Then you can check for nameOfString.isEmpty()